virtool/faker/wrapper.py

```diff
@@ -0,0 +1,10 @@
+from faker import Faker
+from faker.providers import misc
+
+class FakerWrapper(Faker):
+    fake = Faker()
+    Faker.seed(0)
+    fake.add_provider(misc)
+
+    def get_mongo_id(self):
+        return self.fake.password(length=8, special_chars=False)
```

Comment:
Don't subclass Faker. Not a good idea unless it is intended to be subclassed or you really know its internals.
```suggestion
class FakerWrapper:

    def __init__(self):
        self.fake = Faker()
        self.fake.seed_instance(0)
        self.fake.add_provider(misc)

    def get_mongo_id(self):
        return self.fake.password(length=8, special_chars=False)
```